lab_6/task_6_9.py

```diff
@@ -0,0 +1,16 @@
+import io
+
+
+def percent_count():
+    with io.open('C:/Users/Andrii/Desktop/mon_cristo.txt', 'r', encoding='utf-8') as m_cristo:
```

Comment:
you used the same directory path many times.
make it as a constant. move to config file or const file. and reuse it in your tasks